### ISSUE TITLE
Allow opening context menu with shortcut

### ISSUE THREAD

**Issue by OlliTietavainenVaadin** [vaadin/vaadin-context-menu-flow#75]
`vaadin.version 13.0.6`
If you use `contextMenu.setOpenOnClick(true)`, you can make the context menu open on left mouse button clicks. It would then make sense that using `contextMenu.addClickShortcut` afterwards would enable you to open the context menu using the defined shortcut - currently, using the shortcut does nothing.

**Comment by pekam** [comment]
Actually `ContextMenu` should not have this API at all. It exists because `GeneratedVaadinContextMenu implements ClickNotifier`. This doesn't make any sense, because `<vaadin-context-menu>` element itself is not visible (and thus clickable) at all, it's just used to configure when to open the `<vaadin-context-menu-overlay>`. This 

The click shortcut should be added to the `ContextMenu`'s target (assuming the target component has that API).

**Comment by ywuliu** [comment]
actually, we add addClickShortcut to target, it does not work.
we tried more step:
target.addClickListener(x->this.open());
it does not work neither.  
then, we tried following:
this.addOpenedChangeListener(x->System.out.println("HELP MENU OPEN STATUS CHANGED"));
target.addClickListener(x->{System.out.println("TARGET CLICKED"); this.open();});
we found that TARGET CLICKED, but no "HELP MENU OPEN STATUS CHANGE".
in other words, here, this.open() does nothing
